api/v1beta1/cloudstackcluster_webhook_test.go

```diff
@@ -20,30 +20,39 @@ import (
 	"context"
 
 	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var _ = Describe("CloudStackCluster webhooks", func() {
+	const (
```

Comment:
I'm okay with it, but I really don't think consts are necessary like this in test files.

Reply:
They'd certainly be more important in production code, but I've experienced a fair number of test failures (or even worse, tests that don't fail when they should) due to mismatched literals.  If the same literal is used more than once, or is likely to be used again in the future, I like to use a constant or enum.